api/Makefile

```diff
@@ -21,7 +21,7 @@ endef
 
 PROTOC_VERSION := 3.6.1
 PROTOC_PGV_VERSION := v0.3.0-java.0.20200311152155-ab56c3dd1cf9 # Envoy API from master needs changes from master in protoc-gen-validate. Switch to stable once it's released
-GOLANG_PROTOBUF_VERSION := v1.3.2
+GOLANG_PROTOBUF_VERSION := v1.4.3
```

Comment:
Shall we use this PR to also bump protoc version?

Reply:
Also https://github.com/envoyproxy/protoc-gen-validate/releases

Reply:
I did change it on to of yours in this branch just to check CI
https://github.com/kumahq/kuma/tree/chore/upgrade-proto-protoc

Reply:
good point, upgraded